### ISSUE TITLE
Accessibility: Chat text input needs accessible names

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#2444]
## ARIA input fields do not have accessible names

Chat text input.

> When an input field doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. Learn more.

![image](https://user-images.githubusercontent.com/414923/207453232-369368f3-5728-4803-a998-8243413e3fea.png)

Re: #1826
